gradle.properties

```diff
@@ -1,2 +1,3 @@
 group=org.elasticsearch
 version=3.0.0-SNAPSHOT
+luceneVersion=5.4.0-snapshot-1710880
```

Comment:
hmm, for ones like this, is there any way we can follow up by organizing them similar to maven? All the versions that need changing in other places were grouped together with a comment:

https://github.com/elastic/elasticsearch/blob/master/pom.xml#L49-L56

Its pretty important to direct them to other files they will need to change when modifying these versions, otherwise they will hit problems.